app/src/main/res/drawable/home_background.xml

```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Sorry this is a drive by. Should this also have a license

Reply:
This will be replaced when I have the assets from UX. But I can add them now...